service/modules/process_result.py

```diff
@@ -0,0 +1,194 @@
+                raise ValueError(ERROR_ACCESS_401)
+
+            timezone = pytz.timezone('America/Los_Angeles')
+            current_time = datetime.datetime.now(timezone)
+            file_name = 'DBI_permits_' + str(current_time.year) + str(current_time.month) + str(current_time.day)
+            #DBI_permits_YYYYMMDDHHMI_response.csv  where HH = 24 hour clock Mi  = minutes
+            file_name = 'PTS_Export_09_26.csv'
+            result = self.get_result_file(file_name)
+
+            if result == file_name:
+                # dowloaded the result file
+                self.process_file(self.data_file_path + file_name)
+
+                resp.body = json.dumps(jsend.success({'message': file_name, 'responses':len(file_name)}))
+                resp.status = falcon.HTTP_200
+        #pylint: disable=broad-except
+        except Exception as exception:
+            logging.exception('ProcessResultFile.on_get Exception')
+            resp.status = falcon.HTTP_500
+
+            msg_error = ERROR_GENERIC
+            if exception.__class__.__name__ == 'ValueError':
+                msg_error = "{0}".format(exception)
+
+            resp.body = json.dumps(jsend.error(msg_error))
+
+    #pylint: disable=no-self-use,too-many-locals
+    def process_file(self, file_name):
+        """ process the result file """
+        recipients = ''
+        subject = 'CSV export summary'
+        content = '<table><tr><th>Integration Status</th><th>ErrorD</th><th>Formio ID</th><th>Street #</th><th>Street Name</th><th>SFX</th><th>BB Project ID</th></tr>'
+
+        file_handle = open(file_name, 'r')
+        #exported_submissions = self.get_exported_submissions()
+        data_file = open(self.data_file_path + 'exported_submissions.txt', 'r')
+        data_string = data_file.read()
+        exported_submissions = ast.literal_eval(data_string)
+        data_file.close()
+        tracker = {
+            'formio id': [],
+            'status': [],
+            'error': [],
+            'street_num': [],
+            'street_name': [],
+            'sfx': [],
+            'bb_project_id': []
+        }
+        # loop through the result file to create the summary table
+        #file2 = open('tracker.txt', 'w')
+        for line in file_handle:
+            fields = line.split('|')
+            #skip header
+            if fields[0] == 'FORMIO':
+                continue
+            formio_id = fields[0]
+            status = fields[1]
+            if formio_id in exported_submissions:
+                tracker['formio id'].append(formio_id)
+                tracker['status'].append(status)
+                tracker['error'].append(fields[2])
+                #file2.write(str(exported_submissions[formio_id]))
+                street_num = exported_submissions[formio_id]['projectAddressNumber'] if 'projectAddressNumber' in exported_submissions[formio_id] else ''
```

Comment:
can be simplified to street_num = exported_submission[formio_id].get('projectAddressNumber', '')

Reply:
Good idea.  These field gets will be moved to a helper function in the next sprint(epr-246 and epr-253).